README.md

```diff
@@ -0,0 +1,44 @@
+FTS-Keystone
+=====
+
+Keystone authentication support for Kibana 4.4.x
+
+Installation
+-----
+
+Requires a working version of Kibana. The kibana configuration file (/opt/kibana/config/kibana.yml) should be updated
+with the following:
+
+```
+fts-keystone.port: ${keystone_port}
```

Comment:
I think that needs to explicitly state that we need admin port here.

Based on this reference => https://github.com/openstack/keystone/blob/master/etc/policy.json#L107